velox/functions/lib/string/tests/StringImplTest.cpp

```diff
@@ -111,6 +111,70 @@ class StringImplTest : public testing::Test {
         {"\u1E8A", "\u1E8B"},
         {"\u1E8E", "\u1E8F"}};
   }
+
+  static std::vector<std::tuple<std::string, std::string>>
+  getInitcapUnicodeAnySpaceTestData() {
```

Comment:
Character | Unicode | UTF-8 Bytes | Replacement Method
-- | -- | -- | --
Regular Space | U+0020 | 0x20 | str.replace(' ', ...)
Non-Breaking Space | U+00A0 | 0xC2 0xA0 | str.replace('\u00A0', ' ')

Do you test the 0xC2?